RedHat-UX/red-hat-design-system
# fix(cta): brick full-width

## What I did

1. split up the cta demos into separate files (not strictly related to the issue but i'm kinda *oif shpilkes*, as they say in yiddish, and i was in the files anyways)
2. make brick variant inline-block and full-width always, according to the guidelines.

## Testing Instructions

1. check the demos!
2. check any instances in the wild, see notes below.

## Notes to Reviewers
according to the [guidelines](https://ux.redhat.com/elements/call-to-action/guidelines/), brick variants should always be in a grid, so if this breaks any layouts, the solution should be to fix the container.
